lms/djangoapps/commerce/tests.py

```diff
@@ -184,7 +184,7 @@ def request_callback(_method, _uri, headers):
         # Verify the correct information was passed to the E-Commerce API
         request = httpretty.last_request()
         sku = CourseMode.objects.filter(course_id=self.course.id, mode_slug='honor', sku__isnull=False)[0].sku
-        self.assertEqual(request.body, 'sku={}'.format(sku))
+        self.assertEqual(request.body, '{{"sku": "{}"}}'.format(sku))
```

Comment:
The double braces are needed to escape formatting. Otherwise, `.format()` assumes `sku` is a placeholder instead of static text, and raises an error. See http://stackoverflow.com/a/5466478 (which links to the official Python docs).